dotCMS/src/main/java/com/dotmarketing/servlets/StartupLogger.java

```diff
@@ -0,0 +1,65 @@
+package com.dotmarketing.servlets;
+
+import java.util.TimeZone;
+
+import com.dotcms.enterprise.LicenseUtil;
+import com.dotcms.enterprise.license.LicenseManager;
+import com.dotmarketing.business.CacheLocator;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.business.CacheLocator'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)